Makefile

```diff
@@ -149,20 +149,14 @@ PROTO_SRC_NAMES = $(basename $(notdir $(PROTO_SRCS)))
 PROTO_GO_OUTS = $(foreach name, $(PROTO_SRC_NAMES), go/vt/proto/$(name)/$(name).pb.go)
 
 # This rule rebuilds all the go files from the proto definitions for gRPC.
-proto: $(PROTO_GO_OUTS)
-
+proto: $(PROTO_SRCS)
```

Comment:
I don't think this should take over the proto target. At a minimum we should keep the old target to build the proto's on the local system if the developer would like to

Reply:
Are you able to build those protos? The main reason I started making this change is because I could never get `make proto` to work for me.